lib/Alchemy/Phrasea/TaskManager/Job/IndexerJob.php

```diff
@@ -45,6 +48,17 @@ public function getEditor()
     protected function doJob(JobData $data)
     {
         $app = $data->getApplication();
-        $app['elasticsearch.indexer']->indexScheduledRecords();
+        /** @var Indexer $indexer */
+        $indexer = $app['elasticsearch.indexer'];
+
+        foreach (
+            array_filter(
+                $app->get_databoxes(),
+                function($databox) use($app) {
+                    return $app->is_databox_indexable($databox);
+                }
+            ) as $databox) {
+            $indexer->indexScheduledRecords($databox);
```

Comment:
Should be
```php
$databoxes = array_filter($app->get_databoxes(), function (\databox $databox) use ($app) {
    return $app->is_databox_indexable($databox);
});
$indexer->indexScheduledRecords($databoxes);
```

You can also add a method to `DataboxRepository::findIndexable(): \databox[]`